webpack.config.prod.js

```diff
@@ -4,49 +4,49 @@ const path = require('path');
 const webpack = require('webpack');
 
 module.exports = {
-  entry: {
-    helloWorldPage: 'pages/hello-world/helloWorld.jsx',
-  },
-
-  output: {
-    path: path.resolve(__dirname, 'public'),
-    chunkFilename: 'webpack/[chunkhash].js',
-    filename: 'javascripts/[name].js',
-    publicPath: '/assets/',
-  },
+    entry: {
+        helloWorldPage: 'pages/hello-world/helloWorld.jsx',
```

Comment:
Do you need to add `bundlesLandingPage: 'pages/bundles-landing/bundlesLanding.jsx',` here?